src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  margin: 0 48px 50px 0;
```

Comment:
I used margins because gap on wrapper doesn't work with tests (they don't see gaps)

Reply:
You can add another class to each card 'catalog__card' and give margins to it